pkg/agent/kube.go

```diff
@@ -72,7 +73,7 @@ func (kube *ClusterKubeAPIClient) IsBootstrapConfigMapComplete() (bool, error) {
 	// Get latest version of bootstrap configmap
 	bootstrap, err := kube.Client.CoreV1().ConfigMaps("kube-system").Get(kube.ctx, "bootstrap", v1.GetOptions{})
 
-	if err != nil {
+	if err != nil && !k8serrors.IsNotFound(err) {
```

Comment:
```suggestion
	if !k8serrors.IsNotFound(err) {
```
nit: `IsNotFound` should return false if error is nil

Reply:
It should, but we want this block to _not_ execute if the error is nil.